api/basic/car-created.js

```diff
@@ -54,3 +55,20 @@ export async function updatePinStatus (dynamo, table, cid, status = 'pinned') {
   }))
   return res.Attributes
 }
+
+/**
+ * Send an SQS message witht the S3 path for an S3EventRecord.
+ *
+ * @param {import('aws-lambda').S3EventRecord} record
+ */
+ export async function notifyIndexer (record) {
+  const {
+    SQS_INDEXER_QUEUE_URL: queueUrl,
+    SQS_INDEXER_QUEUE_REGION: region,
+    // set for testing
+    SQS_ENDPOINT: endpoint,
+  } = process.env
+  const sqs = new SQSClient({ region, endpoint })
+  const msg = `${record.awsRegion}/${record.s3.bucket.name}/${record.s3.object.key}`
+  sqs.send(new SendMessageCommand({ QueueUrl: queueUrl, MessageBody: msg }))
```

Comment:
In order for this to work, we'd have to permit this client to put messages on the queue.  That might not be the best design.